<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditAccount</class>
 <widget class="QDialog" name="FormEditAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="m_lblTitle">
       <property name="text">
        <string>URL</string>
       </property>
       <property name="buddy">
        <cstring>m_txtUrl</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="LineEditWithStatus" name="m_txtUrl" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="m_btnTestSetup">
       <property name="text">
        <string>&amp;Test setup</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="m_gbAuthentication">
       <property name="toolTip">
        <string>Some feeds require authentication, including GMail feeds. BASIC, NTLM-2 and DIGEST-MD5 authentication schemes are supported.</string>
       </property>
       <property name="title">
        <string>Authentication</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Username</string>
          </property>
          <property name="buddy">
           <cstring>m_txtUsername</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>m_txtPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="LabelWithStatus" name="m_lblTestResult" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_btnTestSetup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
